terraform/environment/main.tf

```diff
@@ -0,0 +1,61 @@
+module "vm_with_pf_rules_with_ssh_access" {
+  source   = "../modules/vm_with_pf_rules_with_ssh_access"
+
+  vm_name              = "MyVM"
```

Comment:
I would set the base vm_name also from a variable, use `MyVM` by default but users should change the VM name including a new variable like `_VM_BASE_NAME_` or something similar.

Reply:
changed by commit c3ff6d1eca440fc5eaa48cfb16957bfcb0be5091